services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/internal/client/DefaultDynamoDbEnhancedAsyncClientTest.java

```diff
@@ -117,17 +116,4 @@ public void builder_missingDynamoDbClient_shouldCreateDefault() {
         DefaultDynamoDbEnhancedAsyncClient asyncClient = DefaultDynamoDbEnhancedAsyncClient.builder().build();
         assertNotNull(asyncClient.dynamoDbAsyncClient());
     }
-
-    @Test
-    public void createMethod_shouldCreateDefault() {
-        DefaultDynamoDbEnhancedAsyncClient asyncClient = (DefaultDynamoDbEnhancedAsyncClient) DynamoDbEnhancedAsyncClient.create();
-        assertNotNull(asyncClient.dynamoDbAsyncClient());
-    }
-
-    @Test
-    public void toBuilder() {
-        DefaultDynamoDbEnhancedAsyncClient copiedObject = dynamoDbEnhancedAsyncClient.toBuilder().build();
-
-        assertThat(copiedObject, is(dynamoDbEnhancedAsyncClient));
-    }
```

Comment:
I think you removed the wrong test here.

Reply:
Good catch!